scoring_engine/web/templates/overview.html

```diff
@@ -124,7 +125,7 @@ <h4 id="round_start" class="text-center"></h4>
                                                 image_color = 'black';
                                             }
                                             value_str = service_results['host_info']
-                                            html_str = `<span class="glyphicon ${image_icon}" style="color:${image_color}" title="${value_str}"></span>`;
+                                            html_str = `<span class="glyphicon ${image_icon} status" data-clipboard-text="${value_str}" style="color:${image_color}" title="${value_str}"></span>`;
```

Comment:
Should we add something to the bottom of the page, or show some affect when you click on it to copy it?

Reply:
Yeah I'll add some behavior that pops up when you click it

Reply:
This does break hover over for host. Thoughts on which behavior you like more?